components/builder-api/doc/api.raml

```diff
@@ -323,3 +368,29 @@ securitySchemes:
                     description: Project deleted successfully
                 404:
                     description: Project not found
+
+        /jobs:
+            get:
+                description: |
+                    Retrieve the 50 most recently started jobs for the given project.
```

Comment:
We don't do any pagination, sorting, or filtering, either. We decided to do this as a first pass of functionality, because that's all we need at the moment.